portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/lang/en_US.properties

```diff
@@ -30,6 +30,7 @@ users.add.select.with.password=With password
 users.add.select.with.email.and.askPassword=With email verification and ask password from user
 users.add.select.with.otp.and.askPassword=With OTP and ask password from user
 users.add.select.with.email.or.phone=With email/phone verification
+profile=Profile
```

Comment:
It is always good to have long qualified i18n identifiers. This will prevent accidental contamination on strings displayed with i18n. something like users.add.select.with.email.or.phone=...